documentation/helpers/StoryStatus/index.tsx

```diff
@@ -18,9 +18,7 @@ interface Props {
 }
 
 export const StoryStatus = ({ status }: Props): ReactElement | null => {
-  if (status === 'final') {
-    return null
-  }
+  if (status === 'final') return null
```

Comment:
I personally don't like this at all.

It will make it hard to find return statement in the middle of files. I think we should be free to do what we want and have no rule for that.

Reply:
you are free to do as you like ;)
https://eslint.org/docs/latest/rules/curly#multi-line